src/log/access_log.cc

```diff
@@ -475,9 +486,9 @@ fvdbCount(hash_table * hash, const char *key)
 }
 
 void
-fvdbCountVia(const char *key)
+fvdbCountVia(const SBuf &headerValue)
 {
-    fvdbCount(via_table, key);
+    ++TheViaCounts[headerValue];
```

Comment:
Out of scope changes sneaked in again somehow. Whatever recipe you are using to keep your git branch in sync does not work well. This should not be happening often, especially in such a simple PR/context (as far as _merging_ is concerned).